testsuite/tests/ui/test_reset_password.py

```diff
@@ -0,0 +1,39 @@
+"""Test for reset password"""
+
+import re
+
+from testsuite import settings
+from testsuite.ui.views.admin.foundation import BaseAdminView
+from testsuite.ui.views.admin.login import LoginView, ResetAdminPasswordView, ResetPasswordView
+
+
+def test_reset_password(provider_account_user, account_password, mailhog_client, navigator, custom_admin_login):
+    """
+    Test:
+        - Create new provider account
+        - Send reset password email
+        - Reset password
+        - Assert that user is unable to log in with old password
+        - Assert that user is ale to log in with new password
+    """
+    account_name = provider_account_user.entity_name
+    provider_account_user.activate()
+    mail = f"{account_name}@example.com"
+
+    navigator.open(LoginView, url=settings["threescale"]["admin"]["url"])
+    reset_view = navigator.navigate(ResetAdminPasswordView)
+    reset_view.reset_password(mail)
+
+    message = mailhog_client.find_message(subject="Password Recovery", receiver=mail)
+
+    reset_link = re.findall("https://.*\\r", message["items"][0]["Content"]["Body"])[0][:-1]
```

Comment:
when you use `re.search(r"(?P<url>https?://[^\s]+)", message["items"][0]["Content"]["Body"]).group("url")`  you will have only url (not list) without need to remove characters from it